lib/cisco_node_utils/radius_server.rb

```diff
@@ -81,6 +146,7 @@ def default_auth_port
     end
 
     def auth_port=(val)
+      return if platform == :ios_xr
```

Comment:
Not a big fan of a setter behaving as a no-op. Would it be better to raise an error?